config/aws.go

```diff
@@ -53,18 +63,61 @@ type (
 		AWS
 		TableName string `envconfig:"AWS_DYNAMODB_TABLE_NAME"`
 	}
+
+	// ElastiCache holds the basic info required to work with
+	// Amazon ElastiCache.
+	ElastiCache struct {
+		AWS
+		ClusterID string `envconfig:"AWS_ELASTICACHE_CONFIG_ENDPOINT"`
+	}
 )
 
+// MustClient will use the cache cluster ID to describe
+// the cache cluster and instantiate a memcache.Client
+// with the returned from AWS.
```

Comment:
Fixed in https://github.com/NYTimes/gizmo/commit/58839d42150dfebccf93f36e42d407dd0fc13842